cft/format/transform.go

```diff
@@ -94,8 +105,26 @@ func formatNode(n *yaml.Node) *yaml.Node {
 		n.Content[i] = formatNode(child)
 	}
 
-	// Always set Style to 0 for consistent formatting
-	n.Style = 0
+	// Allow global user overrides
+	switch NodeStyle {
+	case "tagged":
+		n.Style = yaml.TaggedStyle
+	case "doublequoted":
+		n.Style = yaml.DoubleQuotedStyle
+	case "singlequoted":
+		n.Style = yaml.SingleQuotedStyle
+	case "literal":
+		n.Style = yaml.LiteralStyle
+	case "folded":
+		n.Style = yaml.FoldedStyle
+	case "flow":
+		n.Style = yaml.FlowStyle
+	case "original":
+		// Do nothing, leave it alone
+	case "":
```

Comment:
if the user makes a typo then wouldn't they just get the default style of yaml ? I think it would be nice if it threw an error in that case. the user used the flag to format yaml a certain way and it would be pretty confusing if they get no feedback for the typo. a default case that panics should take care of it.

Reply:
Done